frontend/lib/src/components/widgets/Selectbox/Selectbox.test.tsx

```diff
@@ -40,18 +41,24 @@ const getProps = (elementProps: Partial<SelectboxProto> = {}): Props => ({
   }),
 })
 
+const pickOption = (selectbox: HTMLElement, value: string): void => {
+  fireEvent.click(selectbox)
+  const valueElement = screen.getByText(value)
+  fireEvent.click(valueElement)
+}
+
 describe("Selectbox widget", () => {
   it("renders without crashing", () => {
     const props = getProps()
-    const wrapper = mount(<Selectbox {...props} />)
-    expect(wrapper.find(UISelect).length).toBeTruthy()
+    render(<Selectbox {...props} />)
+    expect(screen.getByTestId("stSelectboxRoot")).toBeInTheDocument()
```

Comment:
Nit: prefer to follow convention of searching for parent component `stSelectbox` (data-testid already present)

Reply:
done! thanks!